packages/tooltips/src/containers/TooltipContainer.example.md

```diff
@@ -163,7 +165,7 @@ initialState = {
             isVisible
             appendToBody
             placement="top-start"
-            trigger={({ getTriggerProps }) => <TriggerDiv {...getTriggerProps()} />}
+            trigger={({ getTriggerProps, ref }) => <TriggerDiv {...getTriggerProps({ innerRef: ref })} />}
```

Comment:
Plz fix this and later lines for `markdownlint`. https://travis-ci.org/zendeskgarden/react-components/builds/406001331#L1044

Reply:
Weird, my pre-commit hooks didn't run. Will re-apply them. Fixed for now.